src/servicebus/Elsa.AzureServiceBus/Activities/ServiceBusSend.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Elsa;
+using Elsa.Attributes;
+using Elsa.Design;
+using Elsa.Expressions;
+using Elsa.Results;
+using Elsa.Services;
+using Elsa.Services.Models;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Linq;
+using Elsa.AzureServiceBus.Services;
+
+namespace Elsa.AzureServiceBus.Activities
+{
+    [ActivityDefinition(
+        Category = "Azure",
+        Description = "Send a message via azure service bus",
```

Comment:
Descriptions should end with a full stop and use proper capitalization for product names such as Azure Service bus: `"Send a message via Azure Service Bus."`